components/automate-gateway/api/cfgmgmt.pb.swagger.go

```diff
@@ -436,15 +436,15 @@ func init() {
           "description": "Repeated field of dynamically typed values."
         }
       },
-      "description": "` + "`" + `ListValue` + "`" + ` is a wrapper around a repeated field of values.\n\nThe JSON representation for ` + "`" + `ListValue` + "`" + ` is JSON array."
```

Comment:
⚠️ These changes shouldn't be committed, unless we fix the code that generates them.